libp2p-websocket/src/browser.rs

```diff
@@ -191,9 +186,9 @@ impl Transport for BrowserWsConfig {
 			});
 		};
 
-		Ok(open_rx.then(|result| {
+		Ok(Box::new(open_rx.then(|result| {
```

Comment:
Why did we add a `Box` here if it wasn't necessary before?

Reply:
This PR modifies the item produced by `dial`, and writing the type explicitly would become too much of a clusterfuck.

Reply:
Personally I'd write the full type when possible (i.e. when it doesn't use closures) even if it's a clusterfuck but that's me.

Reply:
After looking more at it, I think the problem is that the closure now takes ownership of `original_addr`, which wasn't needed before. Therefore we can either use a closure, or write a custom modifier that makes the code much more difficult to read.